Sources/CoreAardvark/Logging/Logging.swift

```diff
@@ -16,13 +16,27 @@
 
 import Foundation
 
-
+/// Logs a new messages to the default Aardvark log distributor.
+///
+/// - parameter text: The text of the log message.
+/// - parameter type: The type of log message.
+/// - parameter image: An optional image to be attached to the log message. Typically used for messages of type
+/// `.screenshot`.
+/// - parameter parameters: A set of key/value pairs that is persisted with the log message.
+/// - parameter userInfo: A set of key/value pairs that is _not_ persisted with the log message, but rather can be
+/// used to control how the message is processed by various log observers.
```

Comment:
Let's align leading whitespace (note: GitHub UI is treating the spaces I added as tabs – it'll align):
```suggestion
/// - parameter userInfo: A set of key/value pairs that is _not_ persisted with the log message, but rather can be
///                                      used to control how the message is processed by various log observers.
```